README.md

```diff
@@ -3,7 +3,19 @@
 [![license MIT][1]][2] [![code stile: standard][3]][4]
 [![codacy code quality][5]][6] [![codefactor code quality][7]][8]
 
-This repository is dedicated to the study of React design patterns.
+This repository is dedicated to the study of JavaScript and front-end
+development, through the analysis and implementation of React design
+patterns.
+
+From Wikipedia:
+
+> In software engineering, a software design pattern is a general, reusable
+> solution to a commonly occurring problem within a given context in software
+> design. It is not a finished design that can be transformed directly into 
+> source or machine code. Rather, it is a description or template for how to 
+> solve a problem that can be used in many different situations. Design 
```

Comment:
Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/JCPedroza/design-patterns-react/pullRequest?prid=8831468)
```suggestion
> solve a problem that can be used in many different situations. Design
```